files/galaxy/tpv/tool_defaults.yml

```diff
@@ -8,7 +8,10 @@ tools:
     mem: cores * 3.8
     gpus: 0
     env:
-      GALAXY_MEMORY_MB: "{int(mem * 1024)}" # set 5/2023 might be moved to runner or tool wrappers, related to Galaxy issue 15952
+      - name: GALAXY_MEMORY_MB
+        value: "{int(mem * 1024)}" # set 5/2023 might be moved to runner or tool wrappers, related to Galaxy issue 15952
+      - name: HDF5_USE_FILE_LOCKING
```

Comment:
maybe also take over the SINGULARITY env var?

Reply:
Sure! Will push a new commit adding SINGULARITYENV_HDF5_USE_FILE_LOCKING.